thinc/types.py

```diff
@@ -712,7 +711,31 @@ def __get_validators__(cls):
         yield lambda v: validate_array(v, ndim=4, dtype="i")
 
     def __iter__(self) -> Iterator[Ints3d]: ...
-    # def __getitem__(self, key: int) -> Ints3d: ...
+
```

Comment:
These methods aren't actually used anywhere, but Mypy complains if they're not there.